RadarSDK/Radar.m

```diff
@@ -849,4 +873,52 @@ - (void)dealloc {
     [[NSNotificationCenter defaultCenter] removeObserver:self];
 }
 
+
++ (BOOL)isTestKey {
+    NSString *publishableKey = [RadarSettings publishableKey];
+    if ([publishableKey hasPrefix:@"prj_test_pk"] ||
+        [publishableKey hasPrefix:@"org_test_pk"]) {
+        return YES;
+    }
+    return NO;
+}
+
++ (void)sendLog:(RadarLogLevel)level message:(NSString *_Nonnull)message {
+    [[RadarLogBuffer sharedInstance] write:level message:message];
+}
+
+/**
+ * Sends Radar log events to the server
+ */
++ (void)flushLogs {
+    // user _id has to exist
+
+    if (![self isTestKey]) {
+        return;
+    }
+    
+    NSArray<RadarLog *> *flushableLogs = [[RadarLogBuffer sharedInstance] getFlushableLogs];
+    
+    NSUInteger pendingLogCount = [flushableLogs count];
+    if (pendingLogCount == 0) {
+        return;
+    }
+    
+    RadarSyncLogsAPICompletionHandler onSuccess = ^(RadarStatus status){
+        if (status == RadarStatusSuccess) {
+            // clear the logs up to pendinglog count
+            [[RadarLogBuffer sharedInstance] clearSyncedLogsFromBuffer:pendingLogCount];
```

Comment:
Not sure if this will work entirely as expected. Since this is an async operation, the app might log (and purge) more logs before the result comes back - so this could potentially wipe out unexpected logs. Can you instead follow the pattern of removing the logs from the buffer and on error for this callback, add them back to the buffer?